assets/js/updaters.js

```diff
@@ -734,15 +734,16 @@ function updateFissure() {
         timer.attr('data-starttime', moment(fissure.activation).unix());
         timer.attr('data-endtime', moment(fissure.expiry).unix());
       } else {
-        let fissureRow = `<li class="list-group-item list-group-item-borderless" id="${fissure.id}">`;
-        // fissure timer
-        fissureRow += `<span id="fissuretimer${fissure.id}" class="label timer pull-right" data-starttime="${moment(fissure.activation).unix()}" ` +
-                      `data-endtime="${moment(fissure.expiry).unix()}"></span>`;
+        let fissureRow = `<li class="fissure list-group-item list-group-item-borderless" id="${fissure.id}">`;
 
         // fissure body
         fissureRow += `<span class= "fissure-body">${getImage('fissures', {image: fissure.tierNum, title: `Tier ${fissure.tierNum}`, className: 'fissure-icon'})}`;
         fissureRow += `<b>${fissure.node}</b> | ${fissure.missionType} | ${fissure.tier}</span>`;
 
+        // fissure timer
```

Comment:
The timer should remain at the beginning so it doesn't get wrapped down at lower resolutions.

Reply:
Could you provide a screenshot of the problem?  It looks at least functional to me down to 200px width.

Reply:
it's a past problem that i'd prefer avoid happening again. It's hard to explain, and i'll try to see if i can repro again, but i'd prefer we didn't change the order in which they're added if possible.

Reply:
looks like it's working fine now... cool. i'll approve and merge.